packages/recommend-core/src/getTrendingItems.ts

```diff
@@ -49,6 +49,10 @@ export function getTrendingItems<TObject>({
 
   recommendClient.addAlgoliaAgent('recommend-core', version);
 
+  if (!indexName) {
+    return Promise.resolve({ recommendations: [] });
+  }
```

Comment:
We don't have an objectID in the case of `trending-items` so this is the next best thing.